package.json

```diff
@@ -30,6 +30,7 @@
     "@chakra-ui/theme-tools": "^2.1.1",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@storybook/addon-designs": "^7.0.9",
```

Comment:
I think it would be better to leave this for the follow up PR that actually configures and uses the addon. We can keep this PR limited in scope to upgrading the existing storybook dependencies.